src/lib.rs

```diff
@@ -164,6 +164,71 @@ where
     thread_rng().random()
 }
 
+/// Generate a random value in the given range using the thread-local random number generator.
+///
+/// This function is a shortcut for [`thread_rng().gen_range(range)`](Rng::gen_range).
+///
+/// # Examples
+///
+/// ```
+/// let x = rand::range(1u8..=100);
+/// println!("{}", x);
+///
+/// let y = rand::range(0f32..=1e9);
+/// println!("{}", y);
+/// ```
+///
+/// If you're calling `range()` in a loop, caching the generator as in the
```

Comment:
Do you mean `caching the distribution`?

Reply:
This wording is copied from lines 135-136 above.

Reply:
If the range is non-constant, caching the distribution helps. Either way, caching the result of `rand::thread_rng` also helps (slightly).